merp_outgoing_routing/models/stock_location.py

```diff
@@ -31,8 +31,15 @@ def _compute_outgoing_strategy_sequence(self):
 
         res = self.sudo().search([], order='{} {}'.format(
             field, ['asc', 'desc'][int(strategy_order)]))
+        processed = self.env['stock.location']
         for sequence, location in enumerate(res):
+            if location not in self:
+                continue
             location.strategy_sequence = sequence
+            processed |= location
+        remaining_locations = self - processed
+        for remaining in remaining_locations:
+            remaining.strategy_sequence = 1000
```

Comment:
@bizzappdev , why you put sequence equals to 1000 ?

Reply:
yes.. this might be a mistake but we have to set default value and if we have max number in sorting it will be later for records which are not processed. so it will remain in later part of list.

Reply:
if you have better alternative then feel free to suggest.

Reply:
at least it should be:
```suggestion
            remaining.strategy_sequence = len(res)
```

Reply:
changes are done as suggested.